microprofile/WFLY-19592_MicroProfile_Fault_Tolerance_4_1.adoc

```diff
@@ -0,0 +1,127 @@
+---
+categories:
+- microprofile
+stability-level: default
+issue: https://github.com/wildfly/wildfly-proposals/issues/626
+feature-team:
+ developer: rhusar
+ sme:
+  - rhusar
+ outside-perspective:
+  -
+promotes:
+promoted-by:
+---
+= WFLY-19592 [MP Platform 7] MicroProfile Fault Tolerance 4.1
+:author:            Radoslav Husar
+:email:             rhusar@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+Goal is to integrate MicroProfile (MP) Fault Tolerance (FT) 4.1 specification from the MP Platform 7.0 to WildFly at default stability level.
+To certify functionality, the MP FT TCK is run as part of the WildFly testsuite.
+
+=== User Stories
+
+User wants to deploy an application using MicroProfile Fault Tolerance 4.1 from MicroProfile Platform 7.0.
```

Comment:
Would there be a chance to mention a user story that emphasize the main feature in this minor upgrade, i.e. integration with MicroProfile Telemetry generated metrics? 
Just in order to be more specific here. WDYT @rhusar ?